patterns/fargate-serverless/README.md

```diff
@@ -52,31 +52,42 @@ See [here](https://aws-ia.github.io/terraform-aws-eks-blueprints/getting-started
     apiVersion: v1
     data:
       filters.conf: |
-        [FILTER]
-          Name parser
-          Match *
-          Key_Name log
-          Parser regex
-          Preserve_Key True
-          Reserve_Data True
-      flb_log_cw: "true"
-      output.conf: |
-        [OUTPUT]
-          Name cloudwatch_logs
-          Match *
-          region us-west-2
-          log_group_name /fargate-serverless/fargate-fluentbit-logs20230509014113352200000006
-          log_stream_prefix fargate-logs-
-          auto_create_group true
-      parsers.conf: |
-        [PARSER]
-          Name regex
-          Format regex
-          Regex ^(?<time>[^ ]+) (?<stream>[^ ]+) (?<logtag>[^ ]+) (?<message>.+)$
-          Time_Key time
-          Time_Format %Y-%m-%dT%H:%M:%S.%L%z
-          Time_Keep On
-          Decode_Field_As json message
+          [FILTER]
```

Comment:
do we need this here? If its in the addon https://github.com/aws-ia/terraform-aws-eks-blueprints-addons/pull/418 - can we drop it here?

Reply:
Not really sure, I just replaced the old YAML. It's more a validation step. I'll change the command removing the `-o yaml`, to really just check if its there.